packages/rs-drive-abci/src/execution/platform_events/protocol_upgrade/check_for_desired_protocol_upgrade/v0/mod.rs

```diff
@@ -42,13 +48,21 @@ impl<C> Platform<C> {
             ));
         }
 
+        tracing::trace!(
+            total_hpmns = total_hpmns,
+            required_upgraded_hpmns = required_upgraded_hpmns,
+            current_votes = votes_str,
+            versions_passing_threshold = versions_passing_threshold.len(),
+            "Check protocol upgrade: check_for_desired_protocol_upgrade: total_hpmns: {}, required_upgraded_hpmns: {}, current_votes: {}, versions_passing_threshold: {}",
```

Comment:
"Protocol version voting is finished. {} versions passing the threshold: {:?}", versions_passing_threshold.len(), versions_passing_threshold